tests/file-magic-keywords/test.yaml

```diff
@@ -0,0 +1,12 @@
+pcap: ../filestore-filecontainer-http/filecontainer-http.pcap
+
+requires:
+  features:
+    - MAGIC
+
+checks:
+    - filter:
+        count: 1
+        match:
+            event_type: alert
+            alert.signature_id: 1
```

Comment:
Since we're testing for magic, it'll be nice to add that to the checks as well. ;)

Reply:
do you mean adding filemagic: "x" ?